### ISSUE TITLE
react-native-owl not working on Android

### ISSUE THREAD

**Issue by NM-2000** [FormidableLabs/react-native-owl#93]
Tried setting up React-Native-Owl on React-Native Project to capture screenshots on Android. It doesn't work!

**Setup**

- Setup owl.config.json as per the documentation.
```
{
    "android": {
      "packageName": "com.loki",
      "quiet": true
    },
    "debug": true,
    "report": true
  }
```
- npm run owl build -- --platform android - Works fine ✅
- npm run owl test -- --platform android - Fails ❌

**Error**
```
> loki@0.0.1 owl /Users/nikhilmittal/Desktop/Github/loki
> ./node_modules/react-native-owl/dist/cli/index.js "test" "--platform" "android"

[OWL] Starting websocket server.
[OWL] Running tests on android.
[OWL] WebSocket now listening on port 8123.
Sat Jan  1 09:41:00 IST 2022
date: cannot set date: Operation not permitted
Error: Command failed with exit code 1: adb shell date 01010941
    at makeError (/Users/nikhilmittal/Desktop/Github/loki/node_modules/react-native-owl/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/nikhilmittal/Desktop/Github/loki/node_modules/react-native-owl/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  shortMessage: 'Command failed with exit code 1: adb shell date 01010941',
  command: 'adb shell date 01010941',
  escapedCommand: 'adb shell date 01010941',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
```

**Comment by robwalkerco** [comment]
Hi @NM-2000 

Can I just check if you had an Android emulator running, or a debuggable Android device attached?
If so, please can you share the details so that we can attempt to replicate this issue?

Thanks

**Comment by retyui** [comment]
@robwalkerco I faced with this issue too and emulator is runned 

![image](https://user-images.githubusercontent.com/4661784/173227281-b3cdf7e1-1762-4755-a225-c2e916473987.png)

**Comment by retyui** [comment]
When I run that command manually I see the next error:

```tsx
date: cannot set date: Operation not permitted
```

![image](https://user-images.githubusercontent.com/4661784/173227384-f8e68ba6-1f94-4546-8730-347ced17103e.png)

**Comment by retyui** [comment]
Fix:

```
adb root # first
adb shell date 01010941 # now it won't fail
```

**Comment by robwalkerco** [comment]
@retyui brilliant! Thanks for the fix! Would you like to create a small PR for this fix? (the tests will also need updating a little I think, if you add this additional command)

cc @manosim

**Comment by robwalkerco** [comment]
If you don't have chance to make a PR, then I'm sure either @manosim or I will be able to do it soon.

**Comment by robwalkerco** [comment]
Actually @retyui @NM-2000, this issue is already fixed in the v1.0.0 RC `yarn add react-native-owl@next`. The full v1.0.0 should happen soon 😀

Closed by #85
